src/app/shared/components/button/button.component.scss

```diff
@@ -1,21 +1,29 @@
 .app-button {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    padding: 13px 36px;
-    gap: 10px;
-    background: #007298;
-    border-radius: 4px;
-    color: #fff;
-    text-transform: uppercase;
-    cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 13px 36px;
+  gap: 10px;
+  background: #007298;
```

Comment:
Please use the predefined color variables located under /shared/styles/variables/colors.scss
You can add your own color variables , too.